src/olympia/blocklist/views.py

```diff
@@ -132,6 +156,46 @@ def between(ver, min, max):
 
 
 @non_atomic_requests
+def blocklist_json(request):
+    key = 'blocklist:json'
+    cache.add('blocklist:keyversion', 1)
+    version = cache.get('blocklist:keyversion')
+    response = cache.get(key, version=version)
+    if response is None:
+        response = _blocklist_json(request)
+        cache.set(key, response, 60 * 60, version=version)
+    patch_cache_control(response, max_age=60 * 60)
+    return response
```

Comment:
How should I handle ``clear_cache`` callback here? 

https://github.com/mozilla/addons-server/pull/1950/files/7407ac486f4f37609a7e5639408b2ff0b96b1f63#diff-cc4082e617603c9321d8854ba43e0afeR75